.github/workflows/rennovate.json

```
@@ -0,0 +1,10 @@
+{
+  "extends": ["github>balena-io/renovate-config"],
+  "packageRules": [
+    {
+      "matchDatasources": ["npm"],
+      "matchPackageNames": ["cheerio"],
```

```suggestion
      "matchPackageNames": ["cheerio","another-package-we-override"],
```
You can list multiple packages here to make them unmanaged by renovate if you need.

---

I understand that. Check #99 
it seems only one package is needed to be overridden and I have made that change in #99